infra/modules/database/role_manager/role_manager.py

```diff
@@ -11,6 +11,9 @@
 def lambda_handler(event, context):
     if event == "check":
         return check()
+    elif event == "password_ts":
+        connect_as_master_user()
+        return "Succeeded"
```

Comment:
What was the purpose of this change?

Reply:
It is a method of verifying that the master password is accessible without leaving the password in the logs. The `check` function does not make use of the master user.

Reply:
@charles-nava I don't think this change is necessary. the manage operation connects as the master user, so we would already get code coverage there by running `make infra-update-app-database-roles`.